### PR TITLE
154 cohesive colourscheme

### LINKED ISSUE #154: Update colourscheme for cohesive network to allow more groups
As illustrated below most groups are now grey. Not sure how many groups there will be in the final version, there are 12 currently, maybe worth adding a colour ramp, pending feedback from @anguscalder and others. Thoughts on this welcome.

image

### PR DESCRIPTION
**Pull request by mvl22**
Fixes #154.

Screenshot 2024-03-10 at 18 09 40

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Great job, fixes the immediate issue, and I quite like the colours from a quick glance.

File: src/datasets.js [hunk 1]
```diff
@@ -122,10 +122,7 @@ const datasets = {
 				'line-color': [
 					'match',
 					['get', 'group'],
-					1, '#1230b4',
-					2, '#894cf7',
-					3, '#f07984',
-					4, '#fff551',
+					...colourGradient ('#191970', '#b6d0e2', 12)[1],
```
Comment (Robinlovelace) [inline src/datasets.js:125]: :+1: seems nice and generic, although why hardcode 12 groups, can we pull that out from the data @mvl22 ?

**Review by Robinlovelace — APPROVED** [review]

File: src/datasets.js (hunk 1 above)
Comment (mvl22) [reply src/datasets.js:125]: I don't think there's anyway for Javascript to be reading the PMTiles file and work that out.

Comment (Robinlovelace) [reply src/datasets.js:125]: OK, should be fine to hardcode, just need to pre-calculate number of groups. WiP.

**Pull request merged**